quint/src/cliCommands.ts

```diff
@@ -9,12 +9,12 @@
 
 import { existsSync, readFileSync, writeFileSync } from 'fs'
 import JSONbig from 'json-bigint'
-import { basename, resolve } from 'path'
+import { dirname, basename, resolve } from 'path'
 import { cwd } from 'process'
 import chalk from 'chalk'
 
 import {
-  ErrorMessage, Loc, compactSourceMap, parsePhase1, parsePhase2
+  ErrorMessage, Loc, compactSourceMap, parsePhase1, parsePhase1b, parsePhase2
```

Comment:
can we give the parse phases descriptive names? `1b` doesn't mean anything. how about `loadFiles` or something?

Reply:
Yeah, I have renamed all phases